### PR TITLE
Problem: no compile in play-unreal-plugin repo (fix #160)

### LINKED ISSUE #160: Problem: no compile in play-unreal-plugin repo
add demo
make it compiles in unreal engine editor

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: combine api demo + plugin into one project
release two builds with the current names

### CODE REVIEW

**Comment by leejw51crypto** [comment]
this is v0.0.8 source
- migrate code
- modify github actions

**Comment by damoncro** [comment]
`.github` folder in Plugin/play-unreal-plugin still exists.

File: .github/workflows/dev.yml [hunk 1]
```diff
@@ -39,13 +39,13 @@ jobs:
...
       run: make
-
     - name: Build Plugin
-      working-directory: /home/ue4/play-unreal-plugin
+      working-directory: /home/ue4/play-unreal-plugin/Plugins/play-unreal-plugin
       env:
         HOME: /home/ue4
-      run: make RunUAT
+      run: |        
+        echo $PWD
```
Comment (damoncro) [inline .github/workflows/dev.yml:50]: redundant echo

**Comment by leejw51crypto** [comment]
ok, modifying now

**Review by tomtau — COMMENTED** [review]
I don't follow what the original problem is about ? is it that building of the demo was not being tested together with the plugin, hence the demo is moved here?

**Comment by leejw51crypto** [comment]
currently, plugin cannot build without demo project.
by combing demo project
- can check whether plugin compiles
- also check ci build in future release in plugin 

build piple line is the same
- demo project zip file
- plugin zip file

**Comment by leejw51crypto** [comment]
i think, using current paradigm make sense.
close the pr

**Pull request closed without merging**